notebooks/01_data_preprocessing.ipynb

```diff
@@ -22,7 +22,8 @@
     "import scipy as sp\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import glob
</pre><p>Add a warning in the beginning that you only need this once and if you are using annual you can directly access it from here: ..... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ERA5-dashboard-cookbook/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>